include/perfetto/ext/base/BUILD.gn

```diff
@@ -57,6 +57,7 @@ source_set("base") {
     "unix_task_runner.h",
     "utils.h",
     "uuid.h",
+    "version.h",
```

Comment:
Please don't tdo this, this is intentionally separate. Depend on base:version directly.